### PR TITLE
Filter trees on Verify page by tree tag

### LINKED ISSUE #224: Add search by Tags and Species to Tree Verify Page
Ideally we would be able to search by trees so we can get a list/ review specific species.

### PR DESCRIPTION
**Pull request by nmcharlton**
Resolves #224 

User can specify a single text tag by which to filter trees.
This currently only includes tags added in the 'Additional tags' field when verifying a tree.

Screenshot 2020-09-13 at 19 22 50

### CODE REVIEW

File: server/src/controllers/trees.controller.ts [hunk 1]
```diff
@@ -16,9 +16,15 @@ import {
   // del,
   requestBody,
 } from '@loopback/rest';
+import {ParameterizedSQL} from 'loopback-connector';
```
Comment (dadiorchen) [inline server/src/controllers/trees.controller.ts:19]: I thought you are pretty professional now in terms of Loopback, :D

File: server/src/models/index.ts [hunk 2]
```diff
@@ -3,4 +3,4 @@ export * from './planter.model';
 export * from './species.model';
 export * from './adminUser.model';
 export * from './tag.model';
-export * from './treeTag.model';
+export * from './tree-tag.model';
```
Comment (dadiorchen) [inline server/src/models/index.ts]: Looks like a little different from the convention of naming. Why we must name the model name with dash?
And I noticed that you change underscore to dash, we have convention for that?

**Review by dadiorchen — APPROVED** [review]

File: server/src/models/index.ts (hunk 2 above)
Comment (nmcharlton) [reply server/src/models/index.ts]: This is the style used by the Loopback v4 CLI for multi-word models/endpoints.
I thought it easiest to go along with it, but I'm happy to revert back to the old format for consistency.
What do you think?

Comment (nmcharlton) [reply server/src/models/index.ts]: I'll change back to match the style of other filenames and endpoints.
There's nothing worse than inconsistency.

**Pull request merged**